tests/generic_config_updater/test_pfcwd_interval.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import json
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.utilities import wait_until
+from tests.common.config_reload import config_reload
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success, expect_res_success, expect_op_failure
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+
+pytestmark = [
+    pytest.mark.asic('mellanox')
+]
+
+logger = logging.getLogger(__name__)
+
+READ_FLEXCOUNTER_DB_TIMEOUT = 480
+READ_FLEXCOUNTER_DB_INTERVAL = 20
+
+
+@pytest.fixture(scope="module")
+def ensure_dut_readiness(duthost):
+    """
+    Setup/teardown fixture for pfcwd interval config update tst
+
+    Args:
+        duthost: DUT host object
+    """
+    create_checkpoint(duthost)
+
+    yield
+
+    try:
+        logger.info("Rolled back to original  checkpoint")
+        rollback_or_reload(duthost)
+    finally:
+        delete_checkpoint(duthost)
+
+
+def ensure_application_of_updated_config(duthost, value):
+    """
+    Ensures application of the JSON patch config update by verifying field value presence in FLEX COUNTER DB
+
+    Args:
+        duthost: DUT host object
+        value: expected value of POLL_INTERVAL
+    """
+    def _confirm_value_in_flex_counter_db():
+        poll_interval = duthost.shell('redis-cli -n 5 hget FLEX_COUNTER_GROUP_TABLE:PFC_WD POLL_INTERVAL')["stdout"]
```

Comment:
>redis-cli -n 5 [](http://example.com/codeflow?start=39&length=14)

Suggest use `sonic-db-cli PFC_WD_DB`, which is flexible together with multi-DB or multi-ASIC feature.